itou/eligibility/models/geiq.py

```diff
@@ -117,7 +121,8 @@ def clean(self):
         # The following would have been nice in a unique constraint,
         # but infortunately functions.Now() is not immutable
         if (
-            self.job_seeker
+            job_seeker
+            and author_geiq
```

Comment:
Pour moi, on excluait volontairement la recherche de diagnostics émis par un GEIQ lorsqu’on crée un diag prescripteur ?

Reply:
je n'aurais en effet pas du mettre ici `and author_geiq` parce qu'il peut être None.
Par contre, c'est mystérieux, j'avais ajouté ces variables locales parce que j'avais parfois `self.job_seeker` et `self.author_geiq` qui levait des `RelationDoesNotExists` (ou un truc du genre) et je ne reproduis plus.

Possiblement j'ai corrigé de la bonne manière en continuant à fignoler mon formulaire d'admin.

Reply:
Ok, c'est magique : sur la CI j'ai bin le test qui plante avec un `RelatedObjectDoesNotExist` : https://github.com/gip-inclusion/les-emplois/actions/runs/12943599473/job/36103193034?pr=5439
En local quand je mets un breakpoint ici, j'ai aussi l'erreur, mais le test passe quand même O_O